src/plugins/telemetryTable/pluginSpec.js

```diff
@@ -46,12 +48,24 @@ describe("the plugin", () => {
         tablePlugin = new TablePlugin();
         openmct.install(tablePlugin);
 
+        openmct.install(openmct.plugins.UTCTimeSystem());
```

Comment:
createOpenMCT() already installs this plugin, doesn't it? Is this because you're defining a new start and end time? Maybe extend createOpenMCT to accept time bounds as optional parameters?

Reply:
Oh! Yup, you're right, I forgot :) Thanks, I'll change it.